### PR TITLE
[KYUUBI #1384] Introduce spotless and google java format for java code

### PR DESCRIPTION
**Pull request by yanghua**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1260,6 +1261,34 @@
...
+                            <includes>
+                                <include>src/main/java/**/*.java</include>
+                                <include>src/test/java/**/*.java</include>
+                            </includes>
+
+                            <googleJavaFormat>
+                                <version>1.7</version>
+                                <style>GOOGLE</style>
+                                <reflowLongStrings>false</reflowLongStrings>
+                                <groupArtifact>com.google.googlejavaformat:google-java-format</groupArtifact>
```
Comment (pan3793) [inline pom.xml:1279]: Remove L1278-1279, it's default value

**Comment by pan3793** [comment]
Rebase master, and format the Java code to make check pass.

**Comment by yanghua** [comment]
> Rebase master, and format the Java code to make check pass.

will address later

**Review by yaooqinn — APPROVED** [review]
lgtm

File: pom.xml (hunk 1 above)
Comment (pan3793) [reply pom.xml:1338]: Remove L1337-1338

File: kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiPreparedStatement.java [hunk 2]
```diff
@@ -621,10 +614,9 @@ public void setObject(int parameterIndex, Object x) throws SQLException {
     } else {
       // Can't infer a type.
       throw new SQLException(
-          MessageFormat
-              .format(
-                  "Can''t infer the SQL type to use for an instance of {0}. Use setObject() with an explicit Types value to specify the type to use.",
-                  x.getClass().getName()));
+          MessageFormat.format(
+              "Can''t infer the SQL type to use for an instance of {0}. Use setObject() with an explicit Types value to specify the type to use.",
```
Comment (pan3793) [inline kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiPreparedStatement.java:618]: fix typo

**Review by pan3793 — APPROVED** [review]
2 minor issue, LGTM overall

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**